### ISSUE TITLE
Consider styling the A tags instead of LI tags

### ISSUE THREAD

**Issue by NicholasRBowers** [NicholasRBowers/CSS-Slider-Menu#6]
Both <a> and <li> elements are block elements, and can be styled in the same way.  It may be most prudent to do <a href="#"><li>content</li></a> instead of wrapping the a's with the li's - and then just style the <a> elements.  This is because IE6 doesn't support li:hover for styling, but does support a:hover.

Just a thought.  Not a huge deal.

**Comment by stefanvermaas** [comment]
But as this script is for mobile devices, why would we support IE6? It's not there for mobile...

**Comment by NicholasRBowers** [comment]
Right. But if someone resizes their browser in ie6, they would see it. It
definitely isn't necessary.. just a thought. I'm not sure if it is
considered best practice or not, but I think it is..
